### DISCUSSION TITLE
Does the app update automatically?

### DISCUSSION THREAD

**Discussion by ghost** [noah-nuebling/mac-mouse-fix#120] [Questions] [answered]
Hey all, was just wondering if the app updates automatically. If not, how would one update the app?

**Comment by noah-nuebling** [comment, marked as answer]
Hey @JuNkY14257, thanks for reaching out.
Mac Mouse Fix does support automatic updates!

When the checkbox at “More…” > “Check for Updates”  is checked, the app will check for new versions every time you open the “Mac Mouse Fix” app.

Then, it will ask you want to install the new version.

You can always download the latest stable version on the official website or on the [GitHub releases](https://github.com/noah-nuebling/mac-mouse-fix/releases) page, where you can get an overview of all versions including Beta versions.

Hope that helps and let me know if you have any more questions.

**Reply by ghost** [reply]
Thanks a lot for the info! @noah-nuebling

**Reply by noah-nuebling** [reply]
Sure glad to help :)
